opam-packages-conversion/bin/lib.py

```diff
@@ -234,8 +234,8 @@ def opamRangeToNpmRange(range):
         packageJSON["esy"]["build"] = createPostInstallCommand(
             buildFlatList(d["substs"]), breakList(d["build"]) + breakList(d["install"]))
     packageJSON["dependencies"] = {
-        "substs": "%s/substs" % (config.GH_ORG_NAME,),
-        "opam-installer-bin": "andreypopp/opam-installer-bin",
+        "substs": "esy-ocaml/substs",
+        "opam-installer-bin": "esy-ocaml/opam-installer-bin",
```

Comment:
Is there a reason why we can't use https://github.com/yunxing/opam-installer-bin
?
There's so many orgs/forks I want to try to minimize duplication and start consolidating where possible.